docs/contributors.rst

```diff
@@ -34,11 +34,20 @@ the pull request is merged.
 
 Before a pull request is approved it must meet these criteria:
 
-- unit tests pass
+- Commit messages are descriptive enough
 
-- code coverage from testing does not decrease and new code is covered
+- "Signed-off-by:" line is present in each commit
 
-- new features must have user documentation
+- Unit tests pass
+
+- Code coverage from testing does not decrease and new code is covered
+
+- JSON/YAML configuration changes are updated in the relevant schema
```

Comment:
Is there a way to detect schema changes and catch that they haven't been updated, in the same way that the linter catches format errors and Travis runs the unit tests and helps calculate code coverage?

Reply:
Not really. A configuration change could be something like a new call to `data.get('use_latest')`, and it would be very hard to work out whether that's coming from something a schema is supposed to validate, or whether it is represented in the appropriate schema.

Right now we have unqualified "object" types in JSON Schema definitions, i.e. saying it's an object without saying anything further about what properties it has, so technically the schema definitions can validate but they don't do it as usefully as they could.